src/functionalTest/resources/scenarios/fragments-debug-data/tasks.conf

```diff
@@ -148,50 +138,32 @@ global.handler.fragmentsHandler.config {
         }
       }
     }
-    fetch-payment-providers-fallback {
-      factory = http
-      config.endpointOptions {
-        path = /service/mock/placeholderFallback.json
-        domain = localhost
-        port = ${test.wiremock.mockService.port}
-        allowedRequestHeaders = ["Content-Type"]
-      }
-    }
-    fetch-user-info-fallback {
-      factory = http
-      config.endpointOptions {
-        path = /service/mock/placeholderFallback.json
-        domain = localhost
-        port = ${test.wiremock.mockService.port}
-        allowedRequestHeaders = ["Content-Type"]
-      }
-    }
-    fetch-user-info-fallback-success {
-      factory = http
-      config.endpointOptions {
-        path = /service/mock/placeholderFallback.json
-        domain = localhost
-        port = ${test.wiremock.mockService.port}
-        allowedRequestHeaders = ["Content-Type"]
-      }
-    }
-    fetch-user-info-fallback-error {
-      factory = http
-      config.endpointOptions {
-        path = /service/mock/placeholderFallback.json
-        domain = localhost
-        port = ${test.wiremock.mockService.port}
-        allowedRequestHeaders = ["Content-Type"]
+
+    notify-analytics-cb {
+      factory = cb
+      config {
+        circuitBreakerName = delivery-cb
       }
+      doAction = notify-analytics
     }
-    fetch-user-info-custom-fallback {
+    notify-analytics {
       factory = http
+      config.httpMethod = POST
       config.endpointOptions {
-        path = /service/mock/placeholderFallback.json
+        path = /mock/scenario/analytics
         domain = localhost
-        port = ${test.wiremock.mockService.port}
+        port = ${test.random.brokenService.port}
+        bodyJson = {
+          user-data-id = "{payload.fetch-user-info._result.id}"
```

Comment:
@mmajchrzak This is the example with adding `payload` to the request body, for fragment configuration use `config`.